mne/viz/tests/test_3d.py

```diff
@@ -510,12 +524,97 @@ def test_plot_alignment_surf(renderer):
     )
     _assert_n_actors(fig, renderer, 3)  # left and right hemis plus head
 
+    # surfaces as dict
+    plot_alignment(
+        subject="sample",
+        coord_frame="head",
+        trans=trans_fname,
+        subjects_dir=subjects_dir,
+        surfaces={"white": 0.4, "outer_skull": 0.6, "head": None},
+    )
+
+
+@testing.requires_testing_data
+def test_plot_alignment_surf_errors(renderer, evoked):
+    """Test error raising from plotting of a surface."""
+    info = evoked.info
+    trans = read_trans(trans_fname)
+    # check for correct error if no surf
+    with pytest.raises(RuntimeError, match="No brain surface found"):
+        plot_alignment(
+            info,
+            trans,
+            subject="foo",
+            subjects_dir=subjects_dir,
+            surfaces=["brain"],
```

Comment:
see if you can dedup these param lists by pulling out a shared params dict and overriding / augmenting it as needed for each call.